sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -329,9 +327,9 @@ public void serialize(
 
     private static class AddressResolutionStatistics {
         @JsonSerialize(using = ZonedDateTimeSerializer.class)
-        ZonedDateTime startTime;
+        Instant startTime;
```

Comment:
does `ZonedDateTimeSerializer` now how to serialize Instant?

Reply:
Yes , however renamed the serializer to include instant in name and now it is extending InstantSerializer